docs/docs/guides/manage/console/_add_manager.mdx

```diff
@@ -0,0 +1,6 @@
+When adding a new manager, you can select multiple roles some of which are only allowed to read data.
+This can be especially useful if you add service users for one of your projects where you only need read access.
+
+Per default you will only search for users within the selected organization. If you like to give a role to a user outside the organization you need to switch to the global search and type the exact loginname of the users. This will prevent allowing users to guess users from other organizations.
```

Comment:
```suggestion
Per default you will only search for users within the selected organization. If you like to give a role to a user outside the organization you need to switch to the global search and type the exact loginname of the users. This will prevent users from guessing users from other organizations.
```